src/js/components/RadioButton/doc.js

```diff
@@ -116,6 +116,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '24px',
   },
+  'radioButton.hover.background.color': {
+    description: `The color of the RadioButton border when hovered over.`,
```

Comment:
Think this should say `background` -- but I feel like we need to make this a little more clear that it's going to be the background of the entire div (including the label).

Reply:
I totally forgot to edit that last night just copied from the border Ill fix